packages/common/src/api/account.ts

```diff
@@ -240,14 +225,39 @@ const accountApi = createApi({
   }
 })
 
-export const useGetCurrentUser = () => {
-  const { data: userId } = accountApi.hooks.useGetCurrentUserId({})
-  return useGetUserById({ id: userId! })
+export const useGetCurrentUser = (
```

Comment:
I don't _love_ this pattern, and it's something we have to figure out with the tan-query updates. The idea being that you want a data fetching hook that defers to another hook for most of the work and then selects out a portion of the data or transforms it in some way. We want the wrapped hook to support all the same options and arguments as the original, so we end up doing this `Parameters` thing and modifying the args we care about on the way through.